locales/de.json

```diff
@@ -197,7 +186,7 @@
 				"declarationOfConsentStatus": "Einverständniserklärung Status",
 				"domain": "Domain",
 				"editClass": "Klasse '{{name}}' bearbeiten",
-				"externalExpert": "Externer Experte",
+				"externalExpert": "Externe\/r Expert:in",
```

Comment:
: looks not good for me as Expert is not the male form as Schüler

Reply:
Talked to Konsti about this, and it's "richtig entsprechend Guide".
See Sprachleitfaden: https://docs.hpi-schul-cloud.org/x/nwCS